api-service/src/repo/clubs/map.ts

```diff
@@ -0,0 +1,17 @@
+import { MapToClub } from 'types/club'
+
+export const mapToClub: MapToClub = function (row) {  
+  return {
+    id: row.id,
+    fullName: row.fullname,
+    abbreviation: row.abbreviation,
+    shortName:row.shortName,
+    thorId:row.thorId,
+    regionId:row.regioId,
```

Comment:
should be thorid and regionid (it doesn't come in camelcase from the db)

Reply:
solved